src/js/components.js

```diff
@@ -36,14 +36,55 @@ class User extends React.Component {
 			]);
 	}
 }
+
+class Feedback extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+		}
+		this.handleFeedbackMsg = this.handleFeedbackMsg.bind(this);
+		this.handleFeedbackClass = this.handleFeedbackClass.bind(this);
+	}
+
+	handleFeedbackMsg() {
+		if(this.props.isError) {
```

Comment:
interesting take as you made an entirely separate component
i was thinking along the following:
handleFeedback() {
  let result = {};
  if (this.props.isError) {
    result.message = 'There was a problem...';
    result.additionalClass = 'error';
  } else if ( ..)( {
  } else { }
  return result;
}

if there are classes that should always be there on the component. Better to just explicitly put them into the render routine
i.e.
return e('div', {className: 'feedback ' + (this.state.result && this.state.result.class ? this.state.result.class || ''). id: 'idhere' }, ... );

Reply:
moving the code cleans up the previous component but doesn't address the code duplication IMO, i'm open to discussion as well

Reply:
oh i see what you mean, I was thinking at first of creating an object outside of the component and ended up with creating an entirely seperate component. ill handle the render routine more like you suggested in the later

Reply:
okay then would it better if i just handle it as an object inside the component like you mentioned first?